src/RegistrationController.js

```diff
@@ -128,15 +128,19 @@ function (
       });
       this.model = new Model();
 
-      var checkPasswordMeetComplexities = function(model) {
+      var checkPasswordMeetComplexities = function (model, showError) {
         var password = model.get('password') || '';
         _.each(schema.passwordComplexity.enabledComplexities, function(complexityName) {
-          var ele = Okta.$('.password-complexity-' + complexityName);
+          var ele = Okta.$('#password-complexity-' + complexityName);
           var complexityValue = this.get(complexityName);
+          ele.removeClass();
           if (PasswordComplexityUtil.complexities[complexityName].doesComplexityMeet(complexityValue, password, model)){
-            ele.addClass('password-complexity-meet');
+            ele.addClass('password-complexity-satisfied');
           } else {
-            ele.removeClass('password-complexity-meet');
+            ele.addClass('password-complexity-unsatisfied');
+            if(showError) {
```

Comment:
Also, if you enter this code path, you'll have both ``password-complexity-unsatisfied`` and ``password-complexity-error`` CSS classes applied to the element. Is this what we want?